utils/docker-entrypoint.sh

```diff
@@ -21,11 +21,6 @@ if [[ $AS_BUILDAGENT = "1" ]]; then
 fi
 
 # Manually invoke build process
-# So, first make a shallow clone of the repository if it not exists
-umask 000
-if [ ! -f ./premake5.lua ]; then
-    git clone --depth=1 https://github.com/multitheftauto/mtasa-blue.git .
-fi
```

Comment:
Why? Nothing wrong here. This line allows you to build MTA without cloning the MTA repository.

Reply:
this line actually clones the entire repo all the time.

Reply:
It doesn't. Check your command line and current folder

Reply:
It does. Docker pulls mta repo from github if that line is saved. Without it, it runs only local changes

Reply:
No. I tested it

Reply:
Me too. It didnt accept local copy unless I remove that line.
If you try to build mta then you already have necessary files in your filesystem.

Reply:
```sh
cd <mta sa repo here>
docker build -t mtasabuilder:latest .
docker run --rm -v ./:/build mtasabuilder:latest
```
Does it work?

Reply:
> volume name is too short, names should be at least two alphanumeric characters.

Reply:
```
docker run --rm -v <full path to repo>:/build mtasabuilder:latest
```
?